dependencies.yaml

```diff
@@ -0,0 +1,84 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5"]
+      arch: [x86_64]
+    includes:
+      - build
+      - cudatoolkit
+      - develop
+      - doc
+      - py_version
+      - test_python
+  test_python:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_python
+  test_cpp:
+    output: none
+    includes:
+      - cudatoolkit
+      - test_cpp 
+  checks:
+    output: none
+    includes:
+      - build
+      - develop
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - conda-forge
+  - nvidia
+dependencies:
+  build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cudf=22.12.*
```

Comment:
I would have expected `libcudf` and `librmm` / `rmm` in this list, too? Avoid making those transitive dependencies if cuspatial directly relies on them via `#include` or `cimport`. (If it is only `import`ed in Python, it is a runtime and not a build time dependency.) Also combine this advice with my other comment to split out the C++ and Python build dependencies (we have yet to do this for cudf, still work in progress).